.github/workflows/e2e-aks.yaml

```
@@ -43,8 +43,28 @@ jobs:
           inlineScript: |
             CLUSTER_NAME="operator-e2e-test-${GITHUB_SHA::8}-${{ github.run_number }}"
             echo "CLUSTER_NAME=${CLUSTER_NAME}" >> $GITHUB_OUTPUT
-            az aks create --resource-group ${AZURE_RESOURCE_GROUP} --name "${CLUSTER_NAME}" \
-              --node-count 3 --generate-ssh-keys
+            az aks create \
+              --resource-group=${AZURE_RESOURCE_GROUP} \
+              --name="${CLUSTER_NAME}" \
+              --node-count=3 \
+              --enable-node-public-ip \
+              --generate-ssh-keys > output.json
+            
+            NRG=$(cat output.json | jq -r .nodeResourceGroup)
```

Do we need to create this temporary file? Maybe we can directly use the pipe to `jq`?
```suggestion
            NRG=$(az aks create \
              --resource-group=${AZURE_RESOURCE_GROUP} \
              --name="${CLUSTER_NAME}" \
              --node-count=3 \
              --enable-node-public-ip \
              --generate-ssh-keys | jq -r .nodeResourceGroup)
```

---

Yes, we can simply use pipe to extract `nodeResourceGroup` from output of `az aks create` 👍